aten/src/ATen/ScalarOps.h

```diff
@@ -3,16 +3,16 @@
 #include "ATen/core/Scalar.h"
 #include "ATen/Tensor.h"
 
-namespace at {
+namespace c10 {
```

Comment:
You did this because of ADL, right? Write it down, this is not obvious and you've now got a c10 namespace outside of c10 which is pretty weird.